install_config/aggregate_logging.adoc

```diff
@@ -1518,3 +1518,602 @@ $ oc label node --all logging-infra-fluentd=true <1>
+|Time when the message was received at the collector.
+
+| `pipeline_metadata.normalizer.original_raw_message`
+|The original non-parsed log message as it is received at the normalizer.
+
+| `pipeline_metadata.trace`
+|The field records the trace of the message. Each collector/normalizer appends
+information about itself and the date/time when the message was processed.
+|===
+
+[discrete]
+[[exported-fields-ovirt]]
+=== oVirt Fields
+
+Namespace for oVirt metadata.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `ovirt.entity`
+|The type of the data source, hosts, VMS, and engine.
+
+| `ovirt.host_id`
+|The oVirt host UUID.
+|===
+
+[discrete]
+[[exported-fields-ovirt.engine]]
+=== `ovirt.engine` Fields
+
+Namespace for oVirt engine related metadata. The FQDN of the oVirt engine is
+`ovirt.engine.fqdn`
+
+[discrete]
+[[exported-fields-aushape]]
+=== Aushape Fields
+
+Audit events converted with Aushape. For more information please see
+link:https://github.com/Scribery/aushape[Aushape].
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `aushape.serial`
+|Audit event serial number.
+
+| `aushape.node`
+|Name of the host where the audit event occurred.
+
+| `aushape.error`
+|The error aushape encountered while converting the event.
+
+| `aushape.trimmed`
+|An array of JSONPath expressions relative to the event object, specifying
+objects/arrays with content removed as the result of event size
+limiting. Empty string means event itself. Empty array means trimming occurred
+at unspecified objects/arrays.
+
+| `aushape.text`
+|An array log record strings representing the original audit event.
+|===
+
+[discrete]
+[[exported-fields-aushape.data]]
+=== `aushape.data` Fields
+
+Parsed audit event data related to Aushape.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `aushape.data.avc`
+|type: nested
+
+| `aushape.data.execve`
+|type: string
+
+| `aushape.data.netfilter_cfg`
+|type: nested
+
+| `aushape.data.obj_pid`
+|type: nested
+
+| `aushape.data.path`
+|type: nested
+|===
+
+[discrete]
+[[exported-fields-tlog]]
+=== Tlog Fields
+
+Tlog terminal I/O recording messages. For more information please see
+link:https://github.com/Scribery/tlog[Tlog].
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `tlog.ver`
+|Message format version number.
+
+| `tlog.user`
+|Recorded user name.
+
+| `tlog.term`
+|Terminal type name.
+
+| `tlog.session`
+|Audit session ID of the recorded session.
+
+| `tlog.id`
+|ID of the message within the session.
+
+| `tlog.pos`
+|Message position in the session, milliseconds.
+
+| `tlog.timing`
+|Distribution of this message's events in time.
+
+| `tlog.in_txt`
+|Input text with invalid characters scrubbed.
+
+| `tlog.in_bin`
+|Scrubbed invalid input characters as bytes.
+
+| `tlog.out_txt`
+|Output text with invalid characters scrubbed.
+
+| `tlog.out_bin`
+|Scrubbed invalid output characters as bytes.
+|===
+endif::openshift-origin[]
```

Comment:
@bmcelvee All of the fields listed in  https://github.com/ViaQ/elasticsearch-templates/releases/download/0.0.9/org.ovirt.viaq-collectd.asciidoc should be added